### PR TITLE
Feature/works modal

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

File: src/app/record/components/funding-stack/funding-stack.component.html [hunk 1]
```diff
@@ -23,10 +23,11 @@ <h2 header class="orc-font-body">
     <!-- FUNDING BODY -->
     <app-funding
       *ngIf="stackPanelsDisplay[funding.putCode.value].topPanelOfTheStack"
-      [funding]="funding"
+      [orgDisambiguated]="orgDisambiguated[funding.putCode.value]"
+      [
+      funding]="funding"
```
Comment (amontenegro) [inline src/app/record/components/funding-stack/funding-stack.component.html:28]: Make this a single line

File: src/app/record/components/work-modal/work-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+    )
+    console.log(this.workIdentifiersArray)
+  }
+  deleteWorkId(id: number) {
+    this.workIdentifiersArray.removeAt(id)
+  }
+
+  saveEvent() {
+    this.workForm.markAllAsTouched()
+    console.log('_____')
```
Comment (amontenegro) [inline src/app/record/components/work-modal/work-modal.component.ts:244]: Do we need this console log?

File: src/app/record/components/work-modal/work-modal.component.ts [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+        },
+        workType: {
+          value: this.workForm.value.workType,
+        },
+      }
+      this._workService.save(work).subscribe((value) => {
+        this._dialogRef.close()
+      })
+    } else {
+      console.log(this.workForm.errors)
```
Comment (amontenegro) [inline src/app/record/components/work-modal/work-modal.component.ts:312]: Do we need this? should we show an error to the user?

**Review by amontenegro — CHANGES_REQUESTED** [review]

**Comment by leomendoza123** [comment]
Requested changes are ready

File: src/app/record/components/peer-reviews/peer-reviews.component.html [hunk 4]
```diff
@@ -0,0 +1,279 @@
+<ng-container *ngIf="peerReviews">
```
Comment (DanielPalafox) [inline src/app/record/components/peer-reviews/peer-reviews.component.html:1]: this file was deleted in a previous PR

File: src/locale/properties/record/record.xx.properties [hunk 5]
```diff
@@ -70,3 +70,4 @@ record.selectedItems=X
 record.deleteSelected=X
 record.youHaveNotSelected=X
 record.noItemsSelected=X
+works.countryPublication=X
```
Comment (DanielPalafox) [inline src/locale/properties/record/record.xx.properties:73]: this property should go in the works.en.properties

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by DanielPalafox — APPROVED** [review]

**Pull request merged**